Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/replaceables/externErrors.dfy

```diff
@@ -0,0 +1,23 @@
+// RUN: ! %run %s > %t
+// RUN: %diff "%s.expect" "%t"
+
+replaceable module {:extern "FooNameOverride"} Foo {
+  method {:extern "ZazOverride1"} Zaz() returns (i: int) 
+    ensures i >= 2
+}
+
+module {:extern "BarNameOverride1"} TwoErrors replaces Foo { 
+  // missing error on BarNameOverride2
+  method {:extern "BarNameOverride2", "ZazOverride2"} Zaz() returns (i: int) 
+    ensures i >= 2
+}
+
+replaceable module {:extern "FaaNameOverride"} Faa {
+  method {:extern "ZazOverride1"} Zaz() returns (i: int) 
+    ensures i >= 2
+}
+
+module {:extern} NoErrors replaces Faa { 
+  method {:extern "ZazOverride2"} Zaz() returns (i: int) 
```

Comment:
If I understood your PR description correctly, we're also missing an error here right? If so I think it would be worthwhile to put a comment here too.

Reply:
You are correct ! Fix: https://github.com/dafny-lang/dafny/pull/4889